application/server/notification.go

```diff
@@ -99,7 +97,7 @@ func registerNotifier(c *config.Config, srv types.Server) {
 			logger.Debug().Msg("sending cli path")
 		case sglsp.ShowMessageParams:
 			notifier(c, srv, "window/showMessage", params)
-			logger.Debug().Interface("message", params).Msg("showing message")
+			logger.Debug().Interface("message", params.Message).Msg("showing message")
```

Comment:
Do we still need to use reflection calling Interface here to print the params.Message?